sdk/ml/azure-ai-ml/azure/ai/ml/_internal/_schema/input_output.py

```diff
@@ -39,19 +50,17 @@ class InternalOutputPortSchema(OutputPortSchema):
     datastore_mode = fields.Str()
 
 
+class InternalPrimitiveOutputSchema(OutputPortSchema):
+    type = DumpableEnumField(
+        allowed_values=SUPPORTED_INTERNAL_PARAM_TYPES,
+        required=True,
+    )
+    is_control = fields.Bool()
```

Comment:
What's the behavior if there is a 'mode' specified for primitive output? 'mode' is different from 1.5 to v2, like 'mount' vs 'rw_mount' for output.

Reply:
Interesting... I haven't found any mode specified in component definition, but it does exist on OutputPortSchema

Mode in pipeline node will follow v2 style.